sitemedia/js/controllers/iiif_controller.js

```diff
@@ -97,7 +97,8 @@ export default class extends Controller {
         const url = this.osdTarget.dataset.iiifUrl;
 
         // allow placeholder image (url ending in .png instead of .json)
-        const tileSource = url.endsWith(".png") ? { type: "image", url } : url;
+        const isPlaceholder = url.endsWith(".png");
```

Comment:
this assumes current implementation of iiif only for real images; that's fine with me, just might be worth noting somewhere (although not sure where)